docs/40_initialization.md

```diff
@@ -0,0 +1,454 @@
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Click on `New Org OAuth App` next
+
+<br/><br/>
+<img
+src="../img/new_app.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+You'll be prompted with the following form -- see the example values populated in this screenshot -- more to come in other sections for where these values might be populated:
+
+<br/><br/>
+<img
+src="../img/register_new.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Obtaining your OAuth App Credentials
+
+After creating your OAuth App, you'll lastly want to obtain a client secret key and your client ID -- make sure to note these values for further steps when creating our API
+
+<br/><br/>
+<img
+src="../img/client_values.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Connecting GitHub with Heroku
+
+**This step is contingent that you have forked [dandi-archive](https://github.com/dandi/dandi-archive)**
+
+Once you have the repository forked, you'll want to navigate to `Settings` in the repository page:
+
+<br/><br/>
+<img
+src="../img/linc_settings.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+Next, navigate to `Security > Secrets and variables > Actions`
+
+<br/><br/>
+<img
+src="../img/secrets_actions.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+And then lastly create two new secrets, `HEROKU_EMAIL` should be the email address/account you used to generate your `HEROKU_API_KEY`
+
+<br/><br/>
+<img
+src="../img/heroku_secrets.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+## Terraform Cloud
+
+Terraform is configuration tool for managing "infrastructure-as-code" -- meaning that we can programatically manage infrastructure in
+a traceable, version-controlled form.
+
+The Terraform ecosystem provides a UI tool called **Terraform Cloud**
+
+Start by visiting `https://app.terraform.io/` and making an account.
+
+##### Creating a Terraform Project and Workspace
+
+Once you have successfully made an account, you'll want to create a `Workspace` and a `Project` in that `Workspace`
+
+<br/><br/>
+<img
+src="../img/terraform_init.png"
+alt="retrieve"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+<br/><br/>
+
+##### Populate Environment Variables
+
+After successful creation, you'll want to populate the following variables in the `Variables` section of the `Workspace`:
+
+You'll need to populate `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `HEROKU_API_KEY` and `HEROKU_EMAIL`
```

Comment:
Thanks Aaron.  To simplify, I would suggest removing the phrase `you'll want to` from the instructions.

```suggestion
Populate the following variables in the `Variables` section of the `Workspace`:

1. `AWS_ACCESS_KEY_ID`
2. `AWS_SECRET_ACCESS_KEY`
3. `HEROKU_API_KEY`
4. `HEROKU_EMAIL`
```